packages/shared/src/components/CreateMyFeedButton.tsx

```diff
@@ -26,16 +23,16 @@ const getAnalyticsEvent = (
 export default function CreateMyFeedButton({
   action,
 }: CreateMyFeedButtonProps): ReactElement {
-  const { trackEvent } = useContext(AnalyticsContext);
+  const { trackEvent } = useContext(LogContext);
   const buttonCopy = 'Choose tags';
   const explainerCopy = 'Get the content you need by creating a personal feed';
   const onClick = () => {
-    trackEvent(getAnalyticsEvent('click', buttonCopy));
+    trackEvent(getLogsEvent('click', buttonCopy));
```

Comment:
Wouldn't they be triggered by the `trackEvent` function name?

Reply:
Yes i think they will. Better change to `logEvent`